graphReconstruct/graphReconstruct.go

```diff
@@ -115,7 +115,17 @@ func SetState(node *expandedTree.ETree, position int) {
 		SetState(node.Left, position)
 		SetState(node.Right, position)
 		if allZero(node.Left.Stored) && allZero(node.Right.Stored) {
-			log.Fatal("no Stored values passed to internal node")
+			// switch statement below accounts for potential deletions
+			// if there is a deletion at current position, set Stored list to equal probabilities
+			isLeftDel := node.Left.Fasta.Seq[position]
+			isRightDel := node.Right.Fasta.Seq[position]
+			switch isLeftDel == 10 && isRightDel == 10 {
```

Comment:
I was initially confused what the 10 value was, but I now realize it is the integer value for a gap. It would be best to replace `== 10` with `== dna.Gap` to reduce confusion.

Reply:
I was going to comment on this as well, i agree with dan